participants/File4Ever.md

```diff
@@ -0,0 +1,99 @@
+# <File4Ever>
+
+Before submitting a PR containing your team's project information:
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Registration Submission, Project Name, and Team Slack Handles
+
+*ID: 667253724*
+
+*Project: File4Ever*
+
+*Slack: Steve Tsou*
+
+## (1) Brief Project Description
+
+*A decentralized online personal file warehouse*
+
+## (2) Link to Project UI
+
+*https://fp46hg.axshare.com/#id=03nhrg&p=%E9%A6%96%E9%A1%B5*
+
+## (3) What does your application/UI do?
+
+*Users can upload their personal file onto the app and share or retrieve them any time any where when needed.
+New Users get 32GB storage for 12 months for free.
+Users can pay fil to get more storage or longer service period.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*N/A*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*What sorts of data will you collect?*
+
+*Various.*
+
+*Do you own or have exclusive rights to the data you plan to use?* 
+
+*No. The rights belongs to the owner who uploads the file.*
```

Comment:
Please provide a mechanism for the competition to access the data during the evaluation process - even for personal data.